### PR TITLE
Filled bandoliers (incendiary and lethals)

### PR DESCRIPTION
**Pull request by JoeHammad1844**
## About the PR

Adds two filled bandoliers, one filled with lethals and one filled with incendiaries
gave CBURN a filled incendiary bandolier and removed their ammo boxes

**Media**

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

no cl no fun

### CODE REVIEW

**Comment by metalgearsloth** [comment]
Entities need to have some sort of use otherwise there's no reason to add more.

**Comment by JoeHammad1844** [comment]
ill give CBURN the incendiary one then

File: Resources/Prototypes/Catalog/Fills/Items/belt.yml [hunk 1]
```diff
@@ -148,6 +148,28 @@
...
+- type: entity
+  id: ClothingBeltBandolierFilledIncendiary
+  parent: ClothingBeltBandolier
+  name: bandolier
+  suffix: Incendiary
+  components:
+  - type: StorageFill
+    contents:
+      - id: ShellShotgunIncendiary
+        amount: 60
```
Comment (metalgearsloth) [inline Resources/Prototypes/Catalog/Fills/Items/belt.yml:171]: Don't add 60 entities dog.

**Review by metalgearsloth — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Catalog/Fills/Items/belt.yml (hunk 1 above)
Comment (JoeHammad1844) [reply Resources/Prototypes/Catalog/Fills/Items/belt.yml:171]: but that's the capacity of the bandolier

Comment (metalgearsloth) [reply Resources/Prototypes/Catalog/Fills/Items/belt.yml]: Spawning 30-60 entities for 1 bandolier is really bad and means we have to severely limit how many go in game, just make it use ballisticammoprovider.

Comment (JoeHammad1844) [reply Resources/Prototypes/Catalog/Fills/Items/belt.yml]: ill have to research that one

**Comment by JoeHammad1844** [comment]
I changed it down to 25, which is two shotgun boxes +1, while I figure out how to use ballisticammoprovided in this context

**Comment by metalgearsloth** [comment]
@JoeHammad1844 are you coming back to this.

**Comment by metalgearsloth** [comment]
BallisticAmmoProvider needs doing.

**Comment by JoeHammad1844** [comment]
not sure how to do it

**Comment by JoeHammad1844** [comment]
can i get some guidance?

**Comment by deltanedas** [comment]
is it even possible to have a belt slot item supply ammo to a shotgun

**Comment by metalgearsloth** [comment]
> is it even possible to have a belt slot item supply ammo to a shotgun

Less about supplying ammo to the gun and more about not spawning 25 entities which get carried around.

**Comment by deltanedas** [comment]
having a BAP alt verb to cycle into hand would be good so you dont cycle then turn off combat mode then move and pick it up off the floor

**Comment by JoeHammad1844** [comment]
im a little confused, so inside one bandolier is 25 lethals, inside another is 25 incendiaries, which get put on CBURN

**Comment by JoeHammad1844** [comment]
> having a BAP alt verb to cycle into hand would be good so you dont cycle then turn off combat mode then move and pick it up off the floor

just SHIFT+E

**Comment by deltanedas** [comment]
make it a bap rather than storage whitelisted to shells

**Comment by JoeHammad1844** [comment]
thats out of my pay grade, and I think it works well enough right now because you can have multiple different shells in it and be able to switch between them any time because you just pull out the type you want

**Comment by deltanedas** [comment]
ballistic ammo provider can do that too without spawning 30 entities

**Comment by metalgearsloth** [comment]
Re-open if you come back to this.

**Pull request closed without merging**